src/auth/auth.js

```diff
@@ -254,11 +257,11 @@ router.post('/new-password', async (req, res, next) => {
   res.send('OK');
 });
 
-router.post('/sign-in', async (req, res, next) => {
+router.post('/login', async (req, res, next) => {
```

Comment:
Why the change from `sign-in` to `login`?

Reply:
I want to uniform endpoint names .If we want use `sign-in` rather than `login` it's better to use `sign-up` rather than `register`.
I think:
`sign-up -> sign-in` (is beautiful)
`register -> login` (is beautiful)
but `register -> sign-in` (is a little messy for me :wink:)

Reply:
Good point!